src/status_im/contexts/wallet/create_account/view.cljs

```diff
@@ -50,23 +51,44 @@
 
 (defn- f-view
   []
-  (let [top                          (safe-area/get-top)
-        bottom                       (safe-area/get-bottom)
-        account-color                (reagent/atom (rand-nth colors/account-colors))
-        emoji                        (reagent/atom (emoji-picker.utils/random-emoji))
-        number-of-accounts           (count (rf/sub [:wallet/accounts-without-watched-accounts]))
-        account-name                 (reagent/atom "")
-        placeholder                  (i18n/label :t/default-account-placeholder
-                                                 {:number (inc number-of-accounts)})
-        derivation-path              (reagent/atom (utils/get-derivation-path number-of-accounts))
-        {:keys [public-key address]} (rf/sub [:profile/profile])
-        on-change-text               #(reset! account-name %)
-        primary-name                 (first (rf/sub [:contacts/contact-two-names-by-identity
-                                                     public-key]))
-        {window-width :width}        (rn/get-window)]
+  (let [top                                   (safe-area/get-top)
+        bottom                                (safe-area/get-bottom)
+        account-color                         (reagent/atom (rand-nth colors/account-colors))
+        emoji                                 (reagent/atom (emoji-picker.utils/random-emoji))
+        number-of-accounts                    (count (rf/sub
+                                                      [:wallet/accounts-without-watched-accounts]))
+        account-name                          (reagent/atom "")
+        placeholder                           (i18n/label :t/default-account-placeholder
+                                                          {:number (inc number-of-accounts)})
+        derivation-path                       (reagent/atom (utils/get-derivation-path
+                                                             number-of-accounts))
+        {:keys [address customization-color]} (rf/sub [:profile/profile])
+        on-change-text                        #(reset! account-name %)
+        {window-width :width}                 (rn/get-window)]
```

Comment:
Better not to read subscription's value in the initialization phase of the component, we've had some problems because of this before, e.g. Recently I refactored the whole code to move the subscriptions from the mount phase to the render function in the screen where the token input is being used.

Also using subscriptions inside the render function is not bad, if they aren't going to change, it won't affect, and if they change, we re-render properly

Reply:
BTW, previously a ratom was replaced by a `use-state` hook, and here a ratom is being used, I think another hook should be used instead.

Reply:
@ulisesmac replaced `reagent/atom` usages with `use-state`

Reply:
hmm, I do wonder if we should avoid mixing ratoms and state - probably it's better to just have one type per component and not mix the two.
cc @ilmotta, @cammellos

Reply:
I have too been trying to either use one or the other in the same component, ideally consistent by namespace if it's not too much. Mixing them can get super tricky due to different lifecycles. For better or worse I'm moving to `use-state` and eliminating ratoms where I touch code from now on.

Reply:
@J-Son89 there is no mixing here, I was saying I have replaced `reagent/atoms` with react's `use-state`